starcheat/gui/mainwindow.py

```diff
@@ -57,11 +58,27 @@ def __init__(self):
 
         logging.info("Main window init")
 
+        # check for updates
+        try:
+            latest_tag = request.urlopen("https://github.com/wizzomafizzo/starcheat/releases/latest").geturl()
```

Comment:
`https://github.com/wizzomafizzo/starcheat/releases/latest` redirects always to the latest published tag